src/vm/ClrEtwAll.man

```diff
@@ -6811,6 +6959,9 @@
                 <string id="RundownPublisher.Method.GenericMapMessage" value="Generic" />
                 <string id="RundownPublisher.Method.HasSharedGenericCodeMapMessage" value="HasSharedGenericCode" />
                 <string id="RundownPublisher.Method.JittedMapMessage" value="Jitted" />
+                <string id="RundownPublisher.Method.JitHelperMapMessage" value="JitHelper" />
```

Comment:
It doesn't look like the JitHelper messages are used.

Reply:
It's used in the `MethodFlagsMap`, which is used for the `MethodFlags` fields of method load/unload events